patches/tModLoader/Terraria/NetMessage.cs.patch

```diff
@@ -245,6 +245,15 @@
  							{
  								short num8 = (short)Chest.FindChest(j, i);
  								if (num8 != -1)
+@@ -1625,7 +_,7 @@
+ 									arg_240_0[(int)expr_238] = num10;
+ 								}
+ 							}
+-							if (tile2.type == 425 && tile2.frameX % 36 == 0 && tile2.frameY % 36 == 0)
++							if ((tile2.type == 425 && tile2.frameX % 36 == 0 && tile2.frameY % 36 == 0) || Main.tileSign[tile2.type])
```

Comment:
This looks like the vanilla logic only sends the sign text for one corner of the tile. Should modded signs be different?

Reply:
Hovering over all tiles of the sign, the text of the sign is shown even in multiplayer. This leads me to believe that all tiles share the data of the corner of the tile.

Reply:
I take back what I said. It sends the sign data for each sign tile there is when it should only send data for the one corner. I need to think about this for a while.